kiwi/container/oci.py

```diff
@@ -107,6 +107,13 @@ def __init__(self, root_dir, custom_args=None):
             if 'xz_options' in custom_args:
                 self.xz_options = custom_args['xz_options']
 
+        # for builds inside the buildservice we include a reference to the
+        # specific build. Thus source label only exists inside the
+        # buildservice.
```

Comment:
imho it would be more obvious if the comment which says this is only for the buildservice and the actual code which checks that in Defaults.is_buildservice_worker() would be in the same block of instructions. When I read the comment I needed to search where we the buildservice check appeared. But I leave that up to you

Reply:
yes, you are right, if the _get_source method is private there is no need to check Defaults.is_buildservice_worker inside that method, we can do it out of it.